components/NavigationBar/src/MDCNavigationBar.h

```diff
@@ -98,6 +98,13 @@ IB_DESIGNABLE
  */
 @property(nonatomic, strong, nullable) UIView *titleView;
 
+/**
+ The font applied to the title of navigation bar. If titleTextAttributes is not nil, it will
```

Comment:
What happens if the attributes are provided but don’t have a font specified?

Reply:
We fall into default behavior of UILabel.

Reply:
It may be beneficial to document that this API is ultimately hooking in to a UILabel's font property so that that behavior is well-understood by the client.

With that addition, the "If titleTextAttributes is not nil, it will..." line can probably be removed.

Reply:
Also document what happens when this property is set to nil (because it is null_resettable).